unittest/utility_household.py

```diff
@@ -84,9 +82,9 @@ def consumption(self):
             utility = self.consume_everything()
             assert is_zero(utility - 10 ** 0.2 * 10 ** 0.5 * 10 **
                            0.3), (utility, 10 ** 0.2 * 10 ** 0.5 * 10 ** 0.3)
-            assert self.possession('a') == 0
```

Comment:
This will be hard to port to other language. I suggest to use `self.possession`.

Reply:
True, but in this case I prefer to keep it nice in python and find more awkward springs in other languages later. We don't even know if another language will happen or not.

Reply:
For a standard of an economic simulation library, yes it will be in multiple languages.

Reply:
Ok in this case let's give both options. I will make a pioneered which does not depreciate possessions.